Automattic/wp-calypso
# Add info popover for  tier dropdown

## Proposed Changes

* Add info popover for tier dropdown in lightbox

## Testing Instructions

1. Using the Claypso Green go to the `/pricing` page
2. Open up the AI lightbox (this will work with stats too once [this PR ](https://github.com/Automattic/wp-calypso/pull/90521) is merged and this is rebased)
3. Make sure the info popover is there and that it opens and the content makes sense
![image](https://github.com/Automattic/wp-calypso/assets/65001528/9aaeb74c-363e-4791-a50b-b7c68e1c347c)

## Pre-merge Checklist
